src/main/webapp/app/exercises/shared/feedback/group/programming-feedback-groups.ts

```diff
@@ -61,15 +61,15 @@ class ProgrammingFeedbackGroupInfo extends FeedbackGroup {
     }
 
     shouldContain(feedbackItem: FeedbackItem): boolean {
-        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && (feedbackItem.credits === 0 || !feedbackItem.credits);
+        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && feedbackItem.positive === undefined;
```

Comment:
```suggestion
        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && (feedbackItem.positive === undefined || feedbackItem.credits === 0);
```
I think the case positive == true && credits == 0 should also be in this group.

![grafik](https://user-images.githubusercontent.com/26540346/211197610-92361008-2aa4-47c5-a831-effcf90a7c1b.png)

Reply:
We have to be very careful those predicates, as disjunctivness is not guaranteed. i.e. it could be that feedbacks end up in two groups and are counted multiple times